pages/incometax.vue

```diff
@@ -75,15 +75,27 @@
       <section-title path="incometax.howto.title" />
       <paragraph path="incometax.howto.p1" :margin-bottom="false" />
       <ul>
-        <li
-          v-for="(value, index) in $t('incometax.howto.list')"
-          :key="index"
-          class="mb-1 paragraph-sm xl:paragraph"
-        >
-          {{ value }}
+        <li class="mb-1 paragraph-sm xl:paragraph">
+          <i18n path="incometax.howto.options.o1.content" tag="p" class="paragraph-sm xl:paragraph">
+            <template #video>
+              <a class="text-link" :href="$t('incometax.howto.options.o1.map.video.link')" target="_blank" rel="noopener noreferrer">{{ $t('incometax.howto.options.o1.map.video.value') }}</a>
+            </template>
+          </i18n>
+        </li>
+        <li class="mb-1 paragraph-sm xl:paragraph">
+          {{ $t('incometax.howto.options.o2') }}
         </li>
       </ul>
-      <paragraph path="incometax.howto.p2" :margin-bottom="false" />
+      <paragraph :margin-bottom="false">
+        <i18n path="incometax.howto.p2.content" tag="p" class="paragraph-sm xl:paragraph">
```

Comment:
See my comment [here](https://github.com/communitypoweredbydlot/community-website/pull/59/files#r599431342) as this will cause the same issue and the same fix applies.

Reply:
Done